apps/frontend/src/pages/account/login.tsx

```diff
@@ -101,6 +104,8 @@ export default function Login() {
       <Link to="/map" className="text-primary hover:underline">
         Map
       </Link>
+      <ButtonExample />
+      <LocationDemo />
```

Comment:
Not sure why you put the button example here, also put the location demo in /example route as well.